src/PowerShellEditorServices/Debugging/DebugService.cs

```diff
@@ -57,6 +57,16 @@ public DebugService(PowerShellContext powerShellContext)
         #region Public Methods
 
         /// <summary>
+        /// Returns the passed in path with the [ and ] wildcard characters escaped.
+        /// </summary>
+        /// <param name="path">The path to process.</param>
+        /// <returns>The path with [ and ] escaped.</returns>
+        public static string EscapeWilcardsInPath(string path)
```

Comment:
Also I'd say this method probably belongs in PowerShellContext rather than DebugService since it may have more general utility